src/main/scala/cromwell/binding/AstTools.scala

```diff
@@ -76,6 +77,13 @@ object AstTools {
             k -> v
           }).toMap
           WdlMap(mapType, elements)
+        case a: Ast if a.getName == "ObjectLiteral" && wdlType == WdlObjectType =>
+          val elements = a.getAttribute("map").asInstanceOf[AstList].asScala.toVector.map({ kvnode =>
```

Comment:
Map over a non-anonymous function. I realize it's done this way right above you but I almost certainly made the same comment when that code was written too